### PR TITLE
tiny fixes and update logo and publications

### PR DESCRIPTION
**Pull request by MaxUlysse**
# nf-core/sarek pull request

Many thanks for contributing to nf-core/sarek!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/sarek branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/sarek)
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.md` is updated
- [ ] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Review by FriederikeHanssen — APPROVED** [review]
LGTM

File: nextflow_schema.json [hunk 1]
```diff
@@ -139,51 +139,59 @@
             "fa_icon": "fas fa-cut",
             "properties": {
                 "trim_fastq": {
-                    "type": "string",
+                    "type": "boolean",
                     "fa_icon": "fas fa-cut",
-                    "description": "Run Trim Galore"
+                    "description": "Run Trim Galore",
```
Comment (mashehu) [inline nextflow_schema.json:144]:
```suggestion
                    "description": "Run Trim Galore.",
```

File: nextflow_schema.json [hunk 2]
```diff
@@ -139,51 +139,59 @@
...
+                    "hidden": true
                 },
                 "clip_r1": {
                     "type": "integer",
                     "fa_icon": "fas fa-cut",
                     "description": "Remove bp from the 5' end of read 1",
-                    "help_text": "With Trim Galore"
+                    "help_text": "With Trim Galore",
```
Comment (mashehu) [inline nextflow_schema.json:151]: not a very helpful text, in my opinion. Would easily also fit into the description.

File: nextflow_schema.json [hunk 3]
```diff
@@ -139,51 +139,59 @@
...
+                    "type": "boolean",
                     "fa_icon": "fas fa-cut",
-                    "description": "Run Trim Galore"
+                    "description": "Run Trim Galore",
+                    "hidden": true
                 },
                 "clip_r1": {
                     "type": "integer",
                     "fa_icon": "fas fa-cut",
                     "description": "Remove bp from the 5' end of read 1",
```
Comment (mashehu) [inline nextflow_schema.json:150]:
```suggestion
                    "description": "Number of bp removed from the 5' end of read 1.",
```

File: nextflow_schema.json [hunk 4]
```diff
@@ -139,51 +139,59 @@
...
+                    "help_text": "With Trim Galore",
+                    "hidden": true
                 },
                 "save_trimmed": {
-                    "type": "string",
+                    "type": "boolean",
                     "fa_icon": "fas fa-save",
-                    "description": "Specify how many reads should be contained in the split FastQ file",
-                    "help_text": "If none specified, FastQs won't be split"
+                    "description": "Save trimmed FastQ file intermediates",
```
Comment (mashehu) [inline nextflow_schema.json:185]:
```suggestion
                    "description": "Save trimmed FastQ file intermediates.",
```

File: nextflow_schema.json [hunk 5]
```diff
@@ -139,51 +139,59 @@
...
+                    "help_text": "If none specified, FastQs won't be split",
```
Comment (mashehu) [inline nextflow_schema.json:186]:
```suggestion
                    "help_text": "If none specified, FastQs won't be split.",
```

File: nextflow_schema.json [hunk 6]
```diff
@@ -139,51 +139,59 @@
...
+                    "hidden": true
                 },
                 "split_fastq": {
-                    "type": "string",
-                    "default": "null",
+                    "type": "number",
                     "fa_icon": "fas fa-cut",
-                    "description": "Save trimmed FastQ file intermediates",
-                    "help_text": "If not used, FastQs won't be split"
+                    "description": "Specify how many reads should be contained in the split FastQ file",
```
Comment (mashehu) [inline nextflow_schema.json:192]:
```suggestion
                    "description": "Specify how many reads should be contained in the split FastQ file.",
```

File: nextflow_schema.json [hunk 7]
```diff
@@ -197,14 +205,22 @@
...
+                    "help_text": "> **WARNING** Current indices for `bwa` in AWS iGenomes are not compatible with `bwa-mem2`.\n> Use `--bwa=false` to have `Sarek` build them automatically.",
+                    "hidden": true
                 },
                 "markdup_java_options": {
                     "type": "string",
                     "default": "-Xms4000m -Xmx7g",
                     "fa_icon": "fas fa-memory",
                     "description": "Establish values for GATK MarkDuplicates memory consumption",
-                    "help_text": "See https://github.com/SciLifeLab/Sarek/pull/689"
+                    "help_text": "See https://github.com/SciLifeLab/Sarek/pull/689",
```
Comment (mashehu) [inline nextflow_schema.json:222]:
```suggestion
                    "help_text": "See [https://github.com/SciLifeLab/Sarek/pull/689](https://github.com/SciLifeLab/Sarek/pull/689)",
```

**Comment by MaxUlysse** [comment]
@mashehu thanks for the comments, I'll make another PR with the `.` at the end of the descriptions.

**Pull request merged**